exercises/grade-school/src/School.hs

```diff
@@ -2,10 +2,14 @@ module School (School, add, empty, grade, sorted) where
 
 data School = Dummy
 
-add = undefined
+add :: Int -> String -> School -> School
+add gradeNum student school = undefined
 
+empty :: School
 empty = undefined
 
-grade = undefined
+grade :: Int -> School -> [String]
+grade gradeNum school = undefined
```

Comment:
again, the arg should be something other than `grade`, so I chose `gradeNum` like the example solution.